src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -78,35 +94,65 @@ public async Task<VSReferenceItem[]> HandleRequestAsync(ReferenceParams request,
 
             cancellationToken.ThrowIfCancellationRequested();
 
-            var serverKind = LanguageServerKind.CSharp;
-            var referenceParams = new ReferenceParams()
+            // Temporary till IProgress serialization is fixed
```

Comment:
We should file a tracking issue on our side to revert this once the serialization issue is fixed.

Reply:
My hope is, it's fixed before this is merged. If not, will create the tracking issue.

Not resolving this comment to keep it as a reminder.

Reply:
The external bug fix won't be landing in this sprint. Created: https://github.com/dotnet/aspnetcore/issues/23488